pkg/addons/data/Deployment.apps/openshift-infra/customer-admin-reconciler.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: customer-admin-reconciler
+  namespace: openshift-infra
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: customer-admin-reconciler
+  template:
+    metadata:
+      labels:
+        app: customer-admin-reconciler
+    spec:
+      serviceAccountName: customer-admin-reconciler
+      containers:
+      - args:
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          cat <<'SCRIPT' >/tmp/reconcile
+          #!/bin/bash -x
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          project=$1
+          echo "Project: ${project}"
+
+          # do nothing for whitelisted projects
+          for ns in 'openshift' 'kube-' 'default';
+          do
+            if [ -z "${project##$ns*}" ]; then
+              exit 0
+            fi
+          done
+
+          oc auth reconcile -f /tmp/rolebindings -n "${project}"
+
+          ##### END HERE #######
+          SCRIPT
+          chmod u+x /tmp/reconcile
+
+          exec oc observe namespace --maximum-errors=1 --resync-period=10m -- /tmp/reconcile
```

Comment:
@pweil- @kargakis @jim-minter 
What would be the difference if we `observe project` here instead of `namespace`?  I noticed that namespace gets called 2x on a create where a project only was called a single time.

Reply:
project is a proxy resource for namespace. I don't think it should make a difference to observe projects vs namespaces, it's also probably more correct strictly speaking.